Donor/Entities/Donor.cs

```diff
@@ -28,4 +28,24 @@ public class Donor
     [JsonConverter(typeof(JsonStringEnumConverter))]
     public enum PreferredContactMode { SMS, EMAIL }
 
+    public enum Gender
```

Comment:
Move Gender,BloodGroup  enum into a different file.
Enum and classes can't be in the same file.